### DISCUSSION TITLE
Hello world

### DISCUSSION THREAD

**Discussion by julien-c** [huggingface/awesome-papers#6] [General] [answered]
We now have Discussions – what do you think?

**Comment by joeddav** [comment, marked as answer]
Pretty cool! Going forward, I'll create a thread like this one for each science day and link it from the README.

**Comment by stefan-it** [comment]
Really cool new feature - I'm heavily missing it in Transformers!

**Comment by julien-c** [comment]
Hmm this Discussions redesign looks weird.

**Reply by Pierrci** [reply]
I think I prefer now :)
